### PR TITLE
Reducing node module dependencies 

### PR DESCRIPTION
**Pull request by bhanvimenghani**
This PR contains all the files changed while reducing the number of node module dependencies, like package.json, tsconfig.json etc.

- [x] The React app is working well

Signed-off-by: bhanvimenghani [bmenghan@redhat.com](mailto:bmenghan@redhat.com)

### CODE REVIEW

**Comment by bhanvimenghani** [comment]
This PR corresponds to [issue no. 8](https://github.com/kruize/kruize-ui/issues/8)

File: tsconfig.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "typeRoots": [ "<types-directory-created-in-#1>", "./node_modules/@types"],
```
Comment (bharathappali) [inline tsconfig.json:3]: by default the types in `node_modules/@types` are added, updating `typeRoots` might add other types as well. Can I please know if `<types-directory-created-in-#1>` in `typeRoots`?

**Review by bharathappali — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 1 above)
Comment (bhanvimenghani) [reply tsconfig.json]: Not sure how it was added maybe as a part of some node modules updates. Have tried running the app without it and it works well hence, removed the typeRoots Compiler option.

**Comment by bharathappali** [comment]
LGTM

**Pull request merged**